ui/src/components/TaskDetailsCard/index.jsx

```diff
@@ -214,7 +214,9 @@ export default class TaskDetailsCard extends Component {
                   component="a"
                   href={task.metadata.source}
                   target="_blank"
-                  rel="noopener noreferrer">
+                  rel="noopener noreferrer" 
+                  title ="https://hg.mozilla.org/integration/mozilla-inbound/raw-file/8735151820ebeaf7ee06e9f2ed16868c164d85ab/.taskcluster.yml"
```

Comment:
I think the idea of the issue is that it shows the value of the `source` property for the task -- not that it shows this exact title every time :)

Reply:
I am sorry what do you mean by value of source property for the task.

Reply:
Do you mean the title should appear only when Source is hovered.

Reply:
What you've written will always show the exact same link on hover, for every task.  That's not what we want -- every task will have a different link, and we should show that on hover.